src/remediations/fifi.js

```diff
@@ -392,6 +392,10 @@ function getName (executor) {
         return `Satellite ${executor.id}`;
     }
 
+    if (!smart_management && _.includes([CONNECTED, DISABLED, 'no_rhc'], executor.rhcStatus)) {
```

Comment:
Is there a reason to check if these three values are included instead of just checking that executor.rhcStatus is not null?

Reply:
@Jason-RH good question, yes, because the "no_smart_management" state can't be a marketplace system it has to be part of the check.

Though, I realize now that It would be cleaner to check against NOT being [null, no_smart_management] instead of this.  I'll make that change